test/e2e/bandwidth_test.go

```diff
@@ -98,3 +100,73 @@ func TestBenchmarkBandwidthServiceRemoteAccess(t *testing.T) {
 	skipIfNumNodesLessThan(t, 2)
 	benchmarkBandwidthService(t, masterNodeName(), workerNodeName(1))
 }
+
+func TestPodTrafficShaping(t *testing.T) {
```

Comment:
I assume there is no upstream K8s test for this then?

Reply:
No, there isn't. I can only find unit test about this feature which verifies the bandwidth annotations can be extracted.